pkg/scheduler/apis/config/v1beta3/types.go

```diff
@@ -36,6 +36,8 @@ type LoadAwareSchedulingArgs struct {
 	// When NodeMetrics expired, the node is considered abnormal.
 	// Default is 180 seconds.
 	NodeMetricExpirationSeconds *int64 `json:"nodeMetricExpirationSeconds,omitempty"`
+	// EnableScheduleWhenNodeMetricsExpired Indicates whether nodes with expired nodeMetrics are allowed to schedule pods.
+	EnableScheduleWhenNodeMetricsExpired *bool `json:"enableScheduleWhenNodeMetricsExpired,omitempty"`
```

Comment:
shall we add an argument for the scoring phase?

Reply:
the socre phase will returns 0 when nodeMetric expired, it still has a chance to schedule, so it doesn't seem to have much of an effect.

Reply:
OK